website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -6,11 +6,38 @@ sidebar_label: "MetricFlow time spine"
 ---
 
-MetricFlow uses a timespine table to construct cumulative metrics. By default, MetricFlow expects the timespine table to be named `metricflow_time_spine` and doesn't support using a different name.
+It's common in analytics engineering to have a date dimension or "time spine" table as a base table for different types of time-based joins and aggregations. The structure of this table is typically a base column of daily or hourly dates, with additional columns for other time grains, like fiscal quarter, defined based on the base column. You can join other tables to the time spine on the base column to calculate metrics like revenue at a point in time, or to aggregate to a specific time grain.
 
-To create this table, you need to create a model in your dbt project called `metricflow_time_spine` and add the following code:
+MetricFlow requires you to define a time spine table as a project level configuration, which then is used for various time-based joins and aggregations, like cumulative metrics. 
 
-<File name='metricflow_time_spine.sql'>
+If you already have a date dimension or time spine table in your dbt project you can simply point MetricFlow at this table. To do this, update the `model` configuration to use this table in the semantic layer. For example, given the following directory structure, you can create two time spine configurations, `time_spine_hourly` and `time_spine_daily`.
+
+![Time spine directory structure](/img/docs/building-metrics/time_spines.png)
+
+
+```yaml
+models:
+  - name: time_spine_hourly
+    time_spine:
+      standard_granularity_column: date_hour # column for the standard grain of your table
+    columns:
+      - name: date_hour
+        granularity: hour # set granularity at column-level for standard_granularity_column
+  - name: time_spine_daily
+    time_spine:
+      standard_granularity_column: date_day # column for the standard grain of your table
+    columns:
+      - name: date_day
+        granularity: day # set granularity at column-level for standard_granularity_column
+```
+
+Now, break down the configuration above. It's pointing to a model called `time_spine_daily`. It sets the time spine configurations under the `time_spine` key. The `standard_granularity_column` is the lowest grain of the table, in this case, it's hourly. It needs to reference a column defined under the columns key, in this case, `date_hour`. Use the `standard_granularity_column` as the join key for the time spine table when joining tables in MetricFlow. Here, the granularity of the `standard_granularity_column` is set at the column level, in this case, `hour`.
+
+
+If you need to create a time spine table from scratch, add the following code to your dbt project. 
+The example creates a time spine at a daily grain and an hourly grain. We recommend creating both an hourly and daily time spine, MetricFlow will use the appropriate time spine based on the granularity of the metric selected to minimize data scans.
```

Comment:
Can we add some more detail here? Some things I think it would be helpful to know:
- MetricFlow will use the time spine with the largest compatible granularity for a given query to ensure the most efficient query possible
- You can add a time spine for each granularity you intend to use if minor query efficiency is more important to you than setup time / space constraints
- We recommend having a time spine at the finest grain used in any of your dimensions to avoid unexpected errors